src/form-selector.vue

```diff
@@ -0,0 +1,53 @@
+<script setup>
+import { reactive, onMounted } from 'vue'
+
```

Comment:
I would take all the myKLForms and myLocalForms complications out of the App component and bring a simpler data representation into here:

```js
const myForms = reactive(await Agent.state('my-forms'))

const formIdToName = await Promise.all(
  Object
    .keys(myForms)
    .map(async id => {
      const { name } = await Agent.state(id)
      return [id, name]
    })
   .reduce((acc, [id, name]) => {
      acc[id] = name
      return acc
    }, {})
)
```

that way you can take advantage of the Suspense in App and avoid doing the onmounted loads

Reply:
hmm, I guess you'd need to add a watch on the my-forms scope to update it when things are archived... and then that frees you to do the archiving within the builder component